src/gerrit.js

```diff
@@ -185,15 +186,42 @@ export class Changelist {
     }
 
     if (!this.hasReviewed(user)) {
+      // Check if the latest CL revision is explicitly marked as reviewed or
+      // unreviewed.
+      var current_rev_number = this.getCurrentRevision().getNumber();
+      var latest_tagged_rev_number = -1;
+      var latest_tag_is_reviewed = null;
+      for (const star_label of this.getStars()) {
+        if (star_label.startsWith('reviewed/')) {
+          var reviewed_number = parseInt(star_label.substring(9));
```

Comment:
I don't really like the .substring(...) with a manually computed length.
Can we instead use the following?

```
for (const star_label of this.getStars()) {
  var parts = star_label.split('/');
  if (parts.length != 2)
    continue;
  if (parts[0] != "reviewed" && parts[0] != "unreviewed")
    continue;
  var tagged_rev_number = parseInt(parts[1]);
  if (tagged_rev_number <= latest_tagged_rev_number)
    continue;
  latest_tagged_rev_number = tagged_rev_number;
  latest_tag_is_reviewed = parts[0] == "reviewed";
}
```

Reply:
Done.